### ISSUE TITLE
[BUG] - Pulse sheet viewer cannot be generated.

### ISSUE THREAD

**Issue by GlaserN** [zhinst/laboneq#29]
**Describe the bug**
The following experiment including a realtime sweep on the incremement_oscillator gate cannot be generated with the following error:

```
~/anaconda3/envs/wmilab/lib/python3.10/site-packages/laboneq/pulse_sheet_viewer/pulse_sheet_viewer.py in generate_viewer_html_text(events, title, interactive)
     26     @staticmethod
     27     def generate_viewer_html_text(events, title, interactive: bool = False):
---> 28         events_json = json.dumps(events["event_list"], indent=2)
     29         section_graph_json = json.dumps(events["section_graph"], indent=2)
     30         section_info_json = json.dumps(events["section_info"], indent=2)
...
--> 179         raise TypeError(f'Object of type {o.__class__.__name__} '
    180                         f'is not JSON serializable')
    181 

TypeError: Object of type ParamRef is not JSON serializable
```

[experiment_failed_pulse_sheet.zip](https://github.com/zhinst/laboneq/files/10882480/experiment_failed_pulse_sheet.zip)

**Versions used:**
 - laboneq: 2.1.0

**Comment by QuantumClemens** [comment]
@GlaserN thanks for reporting, we are looking into this

**Comment by QuantumClemens** [comment]
Hi @GlaserN - fix will be deployed with the next release

**Comment by QuantumClemens** [comment]
bug is fixed in version 2.2 - closing this issue
